pkg/operator2/workload/sync_openshift_oauth_apiserver.go

```diff
@@ -86,16 +87,24 @@ func (c *OpenShiftOAuthAPIServerManager) syncOpenShiftOAuthAPIServerDaemonSet(au
 		return nil, err
 	}
 
+	operatorCfg, err := getStructuredConfig(authOperator.Spec.OperatorSpec)
+	if err != nil {
+		return nil, err
+	}
+
+	// log level verbosity is taken from the spec always
+	operatorCfg.APIServerArguments["v"] = []string{loglevelToKlog(authOperator.Spec.LogLevel)}
+	operandFlags := configflags.ToFlagSlice(operatorCfg.APIServerArguments)
+
 	r := strings.NewReplacer(
 		"${IMAGE}", c.targetImagePullSpec,
-		"${OPERATOR_IMAGE}", c.operatorImagePullSpec,
 		// TODO: add LatestAvailableRevision support
 		//"${REVISION}", strconv.Itoa(int(authOperator.Status.LatestAvailableRevision)),
 		"${REVISION}", "1",
-		"${VERBOSITY}", loglevelToKlog(authOperator.Spec.LogLevel),
+		"${FLAGS}", strings.Join(padFlags(operandFlags, strings.Repeat(" ", 14)), " \\\n"),
```

Comment:
isn't there some clever bash escape func we can use?

Reply:
it has to be a valid `YAM` file - I tried to look up a library that would know how to format it properly but didn't find anything.

Reply:
escape for bash and then for yaml 😱

Reply:
https://github.com/alessio/shellescape

Reply:
for yaml escape, we can switch from string replace to yaml replace by unmarshalling first, and then marshalling again.